packages/esm-patient-chart-app/src/visit/visit-form/visit-attribute-type.component.tsx

```diff
@@ -71,13 +60,11 @@ const VisitAttributeTypeFields: React.FC<VisitAttributeTypeFieldsProps> = ({
 };
 
 interface AttributeTypeFieldProps {
+  fieldProps: any;
```

Comment:
Is there any way we can get a sensible type for this?

Reply:
The type for this wasn't exported by the RHF

Reply:
Looks like it's `ControllerRenderProps<TFieldValues, TName>` (see [here](https://github.com/react-hook-form/react-hook-form/blob/ece261ac22762e12eea0e944eb52636e17e61e22/src/types/controller.ts#L88C12-L88C54)).